pom.xml

```diff
@@ -7,6 +7,7 @@
   <groupId>com.elharo.docfix</groupId>
   <artifactId>docfix</artifactId>
   <version>1.0-SNAPSHOT</version>
+  <packaging>maven-plugin</packaging>
```

Comment:
This should not change the existing library artifact. Rather than turning this project into a plugin, it should retain the existing library and introduce a new plugin project that depends on this library. The basic logic should be able to be used by Maven plugins, gradle plugins, and other projects. This PR simply introduces a maven plugin.